infra/src/test/kotlin/fi/fta/geoviite/infra/ui/pagemodel/frontpage/FrontPage.kt

```diff
@@ -22,6 +24,32 @@ class E2EFrontPage : E2EViewFragment(By.className("frontpage")) {
         return E2EPublicationDetailsPage()
     }
 
+    fun openNthSplitPublicationDetails(nth: Int): SplitDetailsDialog {
```

Comment:
Mmh, olen vähän kahden vaiheilla siitä että indeksoidaanko tässä oikeita asioita... Sinällään se kyllä helpottaa laskemista että tässä indeksoidaan julkaisuja joilla ylipäätään on jonkinlainen splitti, mutta se voi myös aiheuttaa jännyyksiä mikäli lukija olettaa (niinkuin itse alunperin oletin) että tässä indeksoitaisiinkin ylipäätään julkaisuja (ensimmäinen splitin sisältävä julkaisuhan voi olla vaikka kolmas julkaisu.)

En tosin nähnyt nopealla vilkaisulla eteenpäin noita itse indeksointeja käytettävän muuten kuin siten, että `nth = 1`. Menisiköhän tämä selkeämmäksi jos näiltä tiputtaisi argumentin kokonaan pois ja muuttaisi nämä aina vaan palauttamaan uusimman splitin sisältävän julkaisun? Ainakin nimistä tulisi ehkä selkeämpiä jos ne saisi muotoon `openLatestSplitDetails` jne.

Reply:
Aa tämänpäs muuttaminen vielä unohtui, tulee jatkofiksinä (laitoin jo mergen tästä).